### PR TITLE
Update root transition operation page

### PR DESCRIPTION
**Pull request by myidpt**
A lot of content in this page are stale and need to be cleaned up:
https://preliminary.istio.io/latest/docs/ops/configuration/security/root-transition/

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ X ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/configuration/security/root-transition/index.md [hunk 1]
```diff
@@ -10,31 +10,22 @@ owner: istio/wg-security-maintainers
 test: n/a
 ---
 
-Istio self-signed certificates have historically had a 1 year default lifetime.
-If you are using Istio self-signed certificates,
+{{< tip >}}
+You can skip this guide if your cluster started with Istio V1.3 or later versions,
```
Comment (frankbu) [inline content/en/docs/ops/configuration/security/root-transition/index.md:14]:
```suggestion
You can skip this guide if your cluster was started with Istio version 1.3 or later,
```

File: content/en/docs/ops/configuration/security/root-transition/index.md [hunk 2]
```diff
@@ -10,31 +10,22 @@ owner: istio/wg-security-maintainers
...
+or if you do not use the Istio self-signed certificates.
+{{< /tip >}}
+
+Before Istio V1.3, the Istio self-signed certificates had a 1 year default lifetime.
+If your cluster started with Istio V1.2 or earlier version,
```
Comment (frankbu) [inline content/en/docs/ops/configuration/security/root-transition/index.md:19]:
```suggestion
Before version 1.3, Istio self-signed certificates had a 1 year default lifetime.
If your cluster started with Istio version 1.2 or earlier,
```

File: content/en/docs/ops/configuration/security/root-transition/index.md [hunk 3]
```diff
@@ -10,31 +10,22 @@ owner: istio/wg-security-maintainers
...
+and it is using Istio self-signed certificates,
 you need to be mindful about the expiration date of the root certificate.
 The expiration of a root certificate may lead to an unexpected cluster-wide outage.
 
 To evaluate the lifetime remaining for your root certificate, please refer to the first step in the
 [procedure below](#root-transition-procedure).
```
Comment (frankbu) [inline content/en/docs/ops/configuration/security/root-transition/index.md:25]:
```suggestion
The expiration of a root certificate may lead to an unexpected cluster-wide outage.
```

File: content/en/docs/ops/configuration/security/root-transition/index.md [hunk 4]
```diff
@@ -10,31 +10,22 @@ owner: istio/wg-security-maintainers
...
-
-If you are not currently using the mutual TLS feature in Istio and will not use it in the future,
-you are not affected and no action is required.
-
-If you may use the mutual TLS feature in the future, you should
-follow the procedure below to perform a root certificate transition.
-
-If you are currently using the mutual TLS feature in Istio with self-signed certificates,
-please follow the procedure and check whether you will be affected.
+The steps below show you how to examine the remaining lifetime for your root certificate,
```
Comment (frankbu) [inline content/en/docs/ops/configuration/security/root-transition/index.md:27]:
```suggestion
The following steps show you how to examine the remaining lifetime for your root certificate,
```

File: content/en/docs/ops/configuration/security/root-transition/index.md [hunk 5]
```diff
@@ -53,43 +44,10 @@ please follow the procedure and check whether you will be affected.
...
-    {{< warning >}}
-    If your Pilot does not have an Envoy sidecar, consider installing one.
-    Pilot has issues using the old root certificate to verify the new workload certificates, which
-    may cause disconnection between Pilot and Envoy.
-    Please see [here](#how-can-i-check-if-pilot-has-a-sidecar) for how to check for this
-    condition.
-    The [Istio upgrade guide](/docs/setup/upgrade/)
-    by default installs Pilot with a sidecar.
-    {{< /warning >}}
+    From Istio V1.5, Envoy can automatically load the new root certificate when it refreshes its
```
Comment (frankbu) [inline content/en/docs/ops/configuration/security/root-transition/index.md:49]:
```suggestion
    Starting with Istio version 1.5, Envoy can automatically load the new root certificate when it refreshes its
```

File: content/en/docs/ops/configuration/security/root-transition/index.md [hunk 6]
```diff
@@ -157,51 +99,7 @@ please follow the procedure and check whether you will be affected.
         }
     {{< /text >}}
 
-    Please inspect the `valid\_from` value of the `ca\_cert`.
-    If it matches the `_Not_ _Before_` value in the new certificate as shown in Step 3,
+    Please inspect the `valid_from` value of `ca_cert`.
```
Comment (frankbu) [inline content/en/docs/ops/configuration/security/root-transition/index.md:102]:
```suggestion
    Inspect the `valid_from` value of `ca_cert`.
```

**Comment by frankbu** [comment]
@myidpt cherrypick label isn't needed on doc PRs until after the release date.

File: content/en/docs/ops/configuration/security/root-transition/index.md (hunk 3 above)
Comment (myidpt) [reply content/en/docs/ops/configuration/security/root-transition/index.md]: Removed line 24.

**Review by frankbu — APPROVED** [review]

**Pull request merged**